tests/test_simulation/test_simulation.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2025 Robotics and AI Institute LLC. All rights reserved.
+
+import numpy as np
+
+from judo.simulation import MJSimulation
+from judo.tasks.cartpole import Cartpole, CartpoleConfig
+from judo.tasks.cylinder_push import CylinderPush, CylinderPushConfig
+
+
+def test_simulation_data_basics() -> None:
+    """Test the simulation data basics."""
+    simulation_data = MJSimulation(init_task="cylinder_push")
+    assert simulation_data.task is not None
+    assert simulation_data.control is None
+    simulation_data.pause()
+    assert simulation_data.paused
+
+
+def test_simulation_data_update_task() -> None:
+    """Test the simulation data update task."""
+    simulation_data = MJSimulation(init_task="cylinder_push")
+    assert isinstance(simulation_data.task.config, CylinderPushConfig)
+    assert isinstance(simulation_data.task, CylinderPush)
+    simulation_data.set_task("cartpole")
+    assert simulation_data.task.nu == 1
+    assert isinstance(simulation_data.task.config, CartpoleConfig)
+    assert isinstance(simulation_data.task, Cartpole)
+
+
+def test_simulation_data_step() -> None:
+    """Test the simulation data step."""
+    np.random.seed(4291)
```

Comment:
nit: you can use the `temp_np_seed` fixture to scope the contents of this test with a temp seed instead of calling `np.random.seed`, which could mess up the global numpy seed state. see `test_controller.py` for usage examples.
```
with temp_np_seed(42):
    # do stuff
```